src/UI/reference-ui/src/createColorMapIconSelector.js

```diff
@@ -1,5 +1,14 @@
 import { IconSelect } from '@thewtex/iconselect.js/lib/control/iconselect'
-import { ColorMapIcons } from 'itk-viewer-color-maps'
+import { ColorMapIcons, CategoricalColorIcons } from 'itk-viewer-color-maps'
+
+const REMOVED_COLOR_MAPS = [
+  'Purples',
+  'Oranges',
+  'PuBu',
+  'BuPu',
+  'BuGn',
+  'GnBu',
+]
```

Comment:
Maybe we should just remove them from itk-viewer-color-maps? What do you think?

Reply:
👍️ Lighten the load a bit.